roles/matrix-base/tasks/main.yml

```diff
@@ -1,18 +1,24 @@
 ---
 
-- ansible.builtin.import_tasks: "{{ role_path }}/tasks/sanity_check.yml"
+- ansible.builtin.import_tasks: "{{ role_path }}/tasks/system_check.yml"
   tags:
     - always
 
+- ansible.builtin.import_tasks: "{{ role_path }}/tasks/sanity_check.yml"
+  tags:
+    - setup-all
+
 - ansible.builtin.import_tasks: "{{ role_path }}/tasks/clean_up_old_files.yml"
   when: run_setup | bool
   tags:
     - setup-all
+    - setup-additional-jvb
```

Comment:
Perhaps `setup-additional-jitsi-jvb` is a better name for this.

I wonder if we should introduce a new tag (called `common` or something of this sort) and use this tag in `matrix-base/tasks/main.yml`, so that we don't need to mention Jitsi stuff.

Later, we can ask people to invoke the "setup additional Jitsi JVB command like this: `ansible-playbook -i inventory/hosts --limit jitsi_jvb_servers setup.yml --tags=common,setup-additional-jitsi-jvb`

Reply:
Yeah I think that's a good idea. 👍